commons/src/main/java/fr/free/nrw/commons/upload/ShareActivity.java

```diff
@@ -30,27 +37,33 @@
     private SingleUploadFragment shareView;
     private CategorizationFragment categorizationFragment;
 
-    public ShareActivity() {
-        super(WikiAccountAuthenticator.COMMONS_ACCOUNT_TYPE);
-    }
-
     private CommonsApplication app;
 
     private String source;
     private String mimeType;
+    private String mediaUriString;
+    private String filePath = "";
+
+    private static final String TAG = "Image";
```

Comment:
Hm, it would still be the tag parameter used by Log. I think the least surprising usage is to use the class name. When the log appears, you know exactly which class to look in. Not a big deal though.